jujugui/static/gui/src/app/components/accordion-section/test-accordion-section.js

```diff
@@ -0,0 +1,96 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2017 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+var juju = {components: {}}; // eslint-disable-line no-unused-vars
+
+chai.config.includeStack = true;
+chai.config.truncateThreshold = 0;
+
+describe('AccordionSection', () => {
+  beforeAll(done => {
+    // By loading this file it adds the component to the juju components.
+    YUI().use('accordion-section', () => {
+      done();
+    });
+  });
+
+  it('can render', () => {
```

Comment:
I'd abstract out the rendering part, perhaps with a render(props) method receiving a props map that can override some internally defined default props. The render function could then return an {instance, output} dict. This way we avoid repeating a very similar code again and again, and allow for more centralised updates when props are modified in the future.